cmd/cmd_suite_test.go

```diff
@@ -20,7 +20,7 @@
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-package cmd
+package cmd_test
```

Comment:
We tend to use the same packages for testing and code, see pitaya here for example:
https://github.com/topfreegames/pitaya/blob/master/acceptor/tcp_acceptor_test.go